cosmoz-data-nav.js

```diff
@@ -925,11 +925,10 @@
 		}
 
 		_onCachePurge(e) {
-			const ids = e.detail.ids;
-			if (!Array.isArray(ids) || ids.length === 0) {
+			if (e === null || e.detail === null || !Array.isArray(e.detail.ids) || e.detail.ids.length === 0) {
```

Comment:
It it more likely that event or event.detail would be undefined rather than null so use `==` instead of `===`

Reply:
Yes, changed it